tox.ini

```diff
@@ -27,6 +29,9 @@ commands =
   python --version
   coverage run test/runtests.py -v --ignore-files='(^\.|^_|^setup\.py$)' \
                                    --ignore-files=not_imported.py \
+                                   pypy: --ignore-files=_mysqldb_test.py \
+                                   pypy: --ignore-files=db_task_history_test.py \
+                                   pypy: --ignore-files=sqla_test.py \
```

Comment:
@Tarrasch I didn't either, I just really hoped it would work and it did.

@erikbern we can't use nose.plugins.attr because it'll fail with an ImportError if it tries to load these files at all due to not installing them. Psycopg2 doesn't work with pypy but it turns out that sqlalchemy and mysql-connector-python do. Updating the PR to remove the ignores.